clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -251,6 +251,19 @@ message Query {
     }
 }
 
+message PluginRescorer {
+    int32 windowSize = 1;
+    Script script = 2;
+}
+
+message QueryRescorer {
+    int32 windowSize = 1;
```

Comment:
If all Rescorer definitions will include the windowSize, it could be moved to the main message.

Reply:
👍